doc/source/programs/pct2rgb.rst

```diff
@@ -39,6 +39,13 @@ RGB file of the desired format.
 
     Generate a RGBA file (instead of a RGB file by default).
 
+.. option:: -pct <palette_file>
+
+    Extract the color table from <palette_file> instead of computing it.
```

Comment:
```suggestion
    Extract the color table from <palette_file> instead of getting it from <source_file>
```

Reply:
Thanks. I've also modified the help string.